docs/scheduler/filter.md

```diff
@@ -117,13 +117,13 @@ name or  ID.  For example, you can start a container called `frontend` running
 `nginx`:
 
 
-    $ docker run -d -p 80:80 --name frontend nginx
+    $ docker run -d -p 80:80 --name front nginx
      87c4376856a8
 
 
     $ docker ps
     CONTAINER ID        IMAGE               COMMAND             CREATED                  STATUS              PORTS                           NODE        NAMES
-    87c4376856a8        nginx:latest        "nginx"             Less than a second ago   running             192.168.0.42:80->80/tcp         node-1      frontend
+    87c4376856a8        nginx:latest        "nginx"             Less than a second ago   running             192.168.0.42:80->80/tcp         node-1      front
 
 
 Then, using `-e affinity:container==frontend` flag schedule a second container to
```

Comment:
I think keeping `frontend` is better, it was changed in https://github.com/docker/swarm/pull/959
Also if we decide to change, we should change everywhere there are still lots of `affinity:com.example.type==frontend`